src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/DetailsRightGrid.tsx

```diff
@@ -24,6 +25,7 @@ import './details-right-grid.scss';
 const DetailsRightGrid: FC = () => {
   const { t } = useKubevirtTranslation();
   const { createModal } = useModal();
+  const cluster = useClusterParam();
```

Comment:
Why not call this directly in `SSHSecretModal`? Is it possible for the VM cluster to be different from the cluster in the params?

Reply:
😆